HHS/OPRE-OPS
# bug: 🐛 starts Alert improvements

## What changed

- adds Alert improvements
- adds modal background for Alert which required redirect
- refactors StepSuccess to StepSuccessAlert
- adds StepSuccessAlert as last step to create Agreements wizard

## Issue

#1536 

## How to test

1. Goto Agreement wizard
2. Complete step 1 and 2
3. Create SCs and BLIs
4. 💰See improved Alerts
5. Finish wizard
6. 💰See improved Alert
7. 💰Profit

## Screenshots

_If relevant, e.g. for a front-end feature_

## Definition of Done Checklist
